docs/developer-guide/dependency-management.md

```diff
@@ -0,0 +1,49 @@
+# Dependency management
+
+Using the dev container should give you access to commands to update poetry which should update the `pyproject.toml` and/or `poetry.lock` files ready to commit to version control.
+
+## Updating build numbers
+
+e.g. `x.y.1` -> `x.y.2`
+
+1. Run `docker compose exec dev poetry update` (Python)
+
+## Major or minor numbers
+
+e.g. `x.1.z` -> `x.2.z` or `1.y.z` -> `2.y.z`
+
+- Update version numbers in `pyproject.toml` (Python)
+- Run `docker compose exec dev poetry update` (Python)
+
+## Adding a dependency
+
+Use the following to automatically use the latest version (e.g. [pendulum](https://pypi.org/project/pendulum/)):
+
+```sh
+docker compose exec dev poetry add DateTime
+docker compose exec dev poetry update
```

Comment:
Its a transfer from ds-wagtail of some bits as is or with some modifications.

I think you had added this to [ds-wagtail](https://github.com/nationalarchives/ds-wagtail/blob/develop/docs/developer-guide/dependency-management.md#adding-a-dependency) a long time ago - perhaps a good practice whenever touching `pyproject.toml` to keep it and `poetry.lock` up to date? 

Re think? I'm happy to remove it - also from other places in this doc. let me know?

Reply:
`poetry add` will update `poetry.lock`.

`poetry update` will update other packages to the latest applicable version. This should ideally be done every so often rather than just when you add new packages.

Reply:
Removed `poetry update`